openbook_auth/checkers.py

```diff
@@ -1357,3 +1357,31 @@ def check_can_get_preview_link_data_for_post(user, post):
         raise ValidationError(
             _('No link associated with post.'),
         )
+
+
+def check_can_subscribe_to_user(subscriber, user):
```

Comment:
can we name these `check_can_subscribe_to_user_notifications`

Following / Subscribing is essentially the same. Let's be specific.